Bookish/Bookish.DataAccess/Library.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using Dapper;
+
+namespace Bookish.DataAccess
+{
+    public class Library
+    {
+        private readonly IDbConnection database = new SqlConnection("Server=localhost;Database=LibraryDB;Trusted_Connection=True;");
```

Comment:
`databaseConnection` is probably more descriptive? And you should have an empty line before the method below 🙂